addons/mail/static/src/utils/common/dates.js

```diff
@@ -4,7 +4,7 @@ const { DateTime } = luxon;
  * @param {luxon.DateTime} datetime
  */
 export function computeDelay(datetime) {
-    if (!datetime) {
+    if (!datetime.isValid) {
```

Comment:
This doesn't look right.
1. Invalid date shouldn't have a delay of 0
2. Why do we have an invalid date now?
3. This is no longer covering empty value.